_config/products.yml

```diff
@@ -592,7 +592,7 @@ jbt_core:
           - name: Release Notes
             url: http://tools.jboss.org/documentation/whatsnew/index.html
       4.3.0.Nightly: 
-        update_site_url: http://download.jboss.org/jbosstools/mars/development/nightly
+        update_site_url: http://download.jboss.org/jbosstools/updates/snapshots/mars/
```

Comment:
the development/nightly seems like bad typo/copy/paste ;/

but why not http://download.jboss.org/jbosstools/mars/nightly/updates/ as we agreed on in chat for the release and used in past ?